pkg/sdk/security_integrations_def.go

```diff
@@ -48,13 +48,47 @@ const (
 	OauthSecurityIntegrationUseSecondaryRolesNone     OauthSecurityIntegrationUseSecondaryRolesOption = "NONE"
 )
 
+var AllOauthSecurityIntegrationUseSecondaryRoles = []OauthSecurityIntegrationUseSecondaryRolesOption{
+	OauthSecurityIntegrationUseSecondaryRolesImplicit,
+	OauthSecurityIntegrationUseSecondaryRolesNone,
+}
+
+func ToOauthSecurityIntegrationUseSecondaryRolesOption(s string) (OauthSecurityIntegrationUseSecondaryRolesOption, error) {
+	s = strings.ToUpper(s)
+	switch s {
+	case "IMPLICIT":
```

Comment:
you can use the values as switch cases like: `string(OauthSecurityIntegrationUseSecondaryRolesImplicit)`

Reply:
Done.